### ISSUE TITLE
bug: log tracker keeps losing track

### ISSUE THREAD

**Issue by CanisUrsa** [ffxiv-teamcraft/ffxiv-teamcraft#2825]
**Describe the bug**
When I just use the desktop app on windows and check off items in the log as I get them (fish/crafting/etc), navigate to a new page and then come back the entire log is reset back to zero.

**To Reproduce**
1. Go to the log tracker, fishing
2. Click on any of the fishing areas and mark a fish as having been completed
3. Navigate to any other place in the application (even within the log tracker like DoH)
4. Navigate back to log tracker, fishing
5. The fish you selected in step 2 will no longer be marked as having been completed

**Expected behavior**
The expectation is that the items marked as logged will stay marked as logged.

**Screenshots**
![ffxivteamcraft_log_bug](https://github.com/ffxiv-teamcraft/ffxiv-teamcraft/assets/37054063/b0fc632d-72cb-4b79-ad3e-8f60358fde8e)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Desktop App
 - Version: 11.1.1

**Comment by Supamiu** [comment]
Yeah I've seen this appear recently due to a `log-tracking` read operation yielding NOT FOUND even if the reqource is actually there.

I can't reproduce this one with just these steps, it seems related to a given set of conditions that the PC has to fullfill and I don't even know which ones just yet.

**Comment by CanisUrsa** [comment]
I run a pihole on my local network that does eat certain URLs. I will attempt the same steps above with the pihole disabled and see if it is related to a domain I have blocked.

Are there any differences between the webpage through Firefox and the desktop app in terms of what domains are utilized? I think the webpage started working late last night but I don't know what happened. I will consider it working if it keeps track for a day or two with multiple browser shutdowns abd computer restarts. I will do more testing with just the webpage version.

**Comment by CanisUrsa** [comment]
Disabling the pihole had no effect on the desktop app. I did notice however that the webpage seems to be working correctly. I am unsure what happened last night but it is keeping track of the fish I tag between restarts, etc. So I have updated this bug to indicate it is only on the desktop app that I am encountering the issue.

If you want me to collection extra data from the desktop application, tell me and I will. Logs, etc.
